help/performance/high-throughput-order-processing.md

```diff
@@ -161,6 +161,21 @@ When disabled, inventory check does not occur when adding a product to the shopp
 
 **Enable Inventory Check On Cart Load** is enabled (set to Yes) by default. To disable the inventory check when loading the cart, set **[!UICONTROL Enable Inventory Check On Cart Load]** to `No` in the Admin UI **Stores** > **Configuration** > **Catalog** > **Inventory** > **Stock Options** section. See [Configure Global Options][global] and [Catalog Inventory][inventory] in the _User Guide_.
 
+## Slave connections
```

Comment:
It may be more appropriate and enticing to use a different title. What about "Load balancing" ?

```suggestion
## Load balancing
```